nextcord/message.py

```diff
@@ -677,6 +682,12 @@ def __init__(
         except AttributeError:
             self.guild = state._get_guild(utils._get_as_snowflake(data, 'guild_id'))
 
+        thread_data = data.get('thread')
+        if self.guild and thread_data:
```

Comment:
Threads are a guild only feature? Why do this check?

Reply:
To convince the type checker (and myself) that it won't break. I can remove it if you want, but I think it's best for safety - Discord may theoretically add threads to DMs in the future, in which case it's better that this code doesn't load them than that it breaks.

Reply:
Honestly it should just break... Doubt they will be adding thread support for DMs